### ISSUE TITLE
PointNet2 discrepancy between model.train() and model.eval()

### ISSUE THREAD

**Issue by richardrl** [pyg-team/pytorch_geometric#1493]
I noticed that, when evaluating my model using PointNet2 class, there is a huge discrepancy between using the model after model.train() and model.eval() when I evaluate a single sample (batch size 1).

It's difficult to produce a minimal repo because it's all wrapped in a private project, but I was wondering if anyone knew why this might be happening.

The model I am using is this one: https://github.com/rusty1s/pytorch_geometric/blob/master/examples/pointnet2_classification.py

In the Net class, with the slight modification that I removed the dropout layers.

I understand there are issues with batchnorm when evaluating on a single sample. But I don't see any batchnorm in that model?

**Comment by rusty1s** [comment]
There is a BatchNorm layer applied in the MLP, which might very well explain those discrepancies. You max want to remove it and test if the issue still remains.
